### DISCUSSION TITLE
群晖 7.0.1 连不上Alist，不知道问题出在哪里？用RaiDrive可以正常挂载打开

### DISCUSSION THREAD

**Discussion by coderdream** [AlistGo/alist#1547] [General]
请大侠帮忙分析一下问题出在哪里？多谢！

Alist 的版本 v2.6.4-0-gbd52df1

![image](https://user-images.githubusercontent.com/1549511/185785024-4ba2d202-3c6b-4fda-a187-ae36f2ee55d5.png)

![image](https://user-images.githubusercontent.com/1549511/185785068-49abdddd-14c1-45e8-a5ed-e9915b221981.png)

![image](https://user-images.githubusercontent.com/1549511/185785082-988d9356-5ad2-47c4-89a0-036f33d1b105.png)

![image](https://user-images.githubusercontent.com/1549511/185785197-0300b518-6fce-4463-a624-4587da311a74.png)

![image](https://user-images.githubusercontent.com/1549511/185785170-5be28810-1983-4288-8c6b-6d8a8a507896.png)

**Comment by bigbigking** [comment]
你还能挂上，我在添加的时候直接提示「连接被服务器拒绝」。
大佬指引说在群晖上需要使用webdavfs访问alist的webdar。
先「在套件中心->设置->套件来源」中增加社区的套件中心，比如添加https://spk7.imnks.com/ (DSM7)或者https://spk.imnks.com/ (DSM6)。
然后搜索安装webdavfs，在按照下面进行操作
https://imnks.com/5952.html
